### ISSUE TITLE
Please update less.php on what less.js version it is based on

### ISSUE THREAD

**Issue by paladox** [oyejorge/less.php#249]
Hi please update less.php to be based on 2.x of less.js release please.

**Comment by oyejorge** [comment]
Would love to... it's a really big undertaking!

**Comment by lkraav** [comment]
Could `less.php` perhaps gain the ability to send LESS files for remote native `less.js` based compilation instead? It'll be an everlasting race to keep up and I doubt this race can be won.

**Comment by paladox** [comment]
Yep I know it would be a big update and I know it wont be done any time soon and probably needs other users to pitch in. And users to test the patch. It may take month, months, years. Just opening this task so that it is a feature suggestion and to be done at some point.

**Comment by oyejorge** [comment]
Remote compilation is a bit different from what we're trying to do.

Hopefully not years. Who knows what version less.js would be at by then if it took us years.
